pkg/rpc/client/rpc.go

```diff
@@ -513,27 +512,23 @@ func (c *Client) CalculateValidUntilBlock() (uint32, error) {
 }
 
 // AddNetworkFee adds network fee for each witness script and optional extra
-// network fee to transaction.
-func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, acc *wallet.Account) error {
+// network fee to transaction. Scripts is an array verification signer's verification scripts.
+// nil means that signer is a contract, otherwise it must be either simple or multisig script.
+func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, scripts ...[]byte) error {
```

Comment:
I'm really unsure it improves anything. Theoretically, you could use `Contract.Parameters` to estimate invocation script size even for non-standard contracts, but changing API this way you lose this ability.

Reply:
Replaced them with accounts.